scripts/filesystem_offload_core_site.xml

```diff
@@ -0,0 +1,31 @@
+<configuration>
```

Comment:
You can place this to the `test` dir,  all the test related file is under the `test` directory.